src/reportengine/resourcebuilder.py

```diff
@@ -188,72 +388,6 @@ def set_result(self, result, spec):
         for action, args in self._node_flags[spec]:
             action(result, self.rootns ,spec, **dict(args))
 
-
-
-    async def _run_parallel(self, deps, completed_spec):
-        try:
-            runnable_specs = deps.send(completed_spec)
-        except StopIteration:
```

Comment:
@Zaharid, I deleted the old _run_parallel, however not so sure to which extent this is a good idea since at the moment the old method seems to deal better with large task as `vp-comparefits`